package.json

```diff
@@ -81,7 +81,7 @@
       "<rootDir>/packages/core/"
     ],
     "testMatch": [
-      "<rootDir>/packages/core/src/__tests__/**/?(*.)(spec|test).ts(x|)"
+      "<rootDir>/packages/core/src/tests/**/?(*.)(spec|test).ts(x|)"
```

Comment:
I would prefer to keep the `__tests__` folder as is, unless you have a reason that justifies the name change?

Reply:
`tests` is shorter to type rather than the awkward `__tests__`. and it makes your paths shorter.

There is also an argument that unit tests should be an essential part of a codebase, so they should follow a normal naming convention rather than an exceptional one.
Also note there are only two `__tests__` folders in this project, so migrating will be pretty easy.

Reply:
Personally, I don't find it awkward (and this is the default for Jest and most JS projects I've seen follow this convention as well). In contrary, I find having a 'test' folder that is alphabetically sorted with the other folders to be more distracting.

Overall, this seems to be a more of a subjective choice than an objective improvement.

Reply:
yaeh, makes sense!

Reply:
I personally prefer to co-locate the tests (for heavily tested code bases)
```
  craft.js/packages/core/src/nodes/Canvas.tsx
  craft.js/packages/core/src/nodes/Canvas.spec.tsx
```

which would resonate with "unit tests should be an essential part of a codebase"

---

In less tested projects it is okay to have them in a separate folder. When keeping the tests in a separate folder I agree with @prevwong - I, too, think it's subjective and personally do not care whether it is `__tests__` or `tests`.